### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by Yaya-Cout**
While I browsed the Upsilon's code, I saw a lot of spelling error, like “Exernal” instead of “External” in the README… In this pull request, I am trying to fix them, but there are probably still some errors because I have 3442 entries in my custom dictionary… If you want, I can upload it. You can find GitHub action build [here](https://github.com/Yaya-Cout/Upsilon/actions/runs/1723616422)

### CODE REVIEW

File: apps/probability/distribution/student_distribution.cpp [hunk 1]
```diff
@@ -33,7 +33,7 @@ double StudentDistribution::cumulativeDistributiveFunctionAtAbscissa(double x) c
   if (std::isinf(x)) {
     return x > 0 ? 1.0 : 0.0;
   }
-  /* TODO There are some computation errors, where the probability falsly jumps to 1.
+  /* TODO There are some computation errors, where the probability falsy jumps to 1.
```
Comment (Tunas1337) [inline apps/probability/distribution/student_distribution.cpp:36]: *falsely

File: apps/apps_container.cpp [hunk 2]
```diff
@@ -47,7 +47,7 @@ AppsContainer::AppsContainer() :
    * poincareCircuitBreaker is run. This means either whitelisting all Epsilon
    * (which makes bigger files to download and slower execution), or
    * whitelisting all the symbols (that's a big amount of symbols to find and
-   * quite painy to maintain).
+   * quite paint to maintain).
```
Comment (Tunas1337) [inline apps/apps_container.cpp:50]: *painful

Comment (Yaya-Cout) [reply apps/apps_container.cpp:50]: Sorry…

**Comment by Tunas1337** [comment]
There's some that made it through. I might, if I'm bored enough, go through this entire commit and fix all false fixes (:stuck_out_tongue_closed_eyes:) at some point.

**Pull request merged**